enhancements/update/automatic-updates.md

```diff
@@ -0,0 +1,176 @@
+
+We might have update edges that are not reversible, so we don't want that A->B->A test to be blocking, but when we decide to stabilize a release candidate where B->A is unstable, we'd want to perform additional verification to convince ourselves that we weren't creating a dead end.
+
+### Graduation Criteria
+
+The ClusterVersion object is already GA, so there would be nothing to graduate or space to cool a preview implementation.
+
+### Upgrade / Downgrade Strategy
+
+The YAML rendering of the old and updated type are compatible, with the only difference being downgrades to earlier releases (where the property was not available) will clear any stored value.
+Upon updating back to a version where the property is available, administrators would need to manually set it to restore automatic update behavior.
+
+### Version Skew Strategy
+
+As a boolean property, this would be [`false` by default][go-zero-values] on updates to the updated ClusterVersion Custom Resource Definition.
+This preserves semantics for existing clusters, where manual updates are the only option.
+
+The installer or administrator could set `automaticUpdates` `true` at cluster-creation time if they wished to have it enabled out of the box on new clusters.
+
+## Implementation History
+
+* [API pull request][api-pull-request].
+
+## Drawbacks
+
+The drawbacks are increased exposure to [automatic-update risks](#risks-and-mitigations).
+
+## Alternatives
+
+A [schedule structure][schedule-proposal] like:
+
+```yaml
+automaticUpdates:
+  schedule:
+  - Mon..Fri *-2..10-* 10:00,14:00
+  - Tues..Thurs *-11..1-* 11:00
+```
+
+The schedule structure was designed to support maintenance windows, allowing for updates on certain days or during certain times.
+But we could enforce maintenance windows on a cluster independently of configuring automatic updates.
+For example, a `schedule` property directly on [the ClusterVersion `spec`][api-spec] would protect administrators from accidentally using the web console or `oc adm upgrade ...` to trigger an update outside of the configured window.
+Instead, administrators would have to adjust the `schedule` configuration or set an override option to trigger out-of-window updates.
+Customization like this can also be addressed by intermediate [policy-engine][cincinnati-policy-engine], without involving the ClusterVersion configuration at all.
+Teaching the cluster-version operator about a local `schedule` filter is effectively like a local policy engine.
+I'm not against that, but it seems orthogonal to an automatic update property.
+
+## Infrastructure Needed
+
+There are already canary clusters will polling automatic updates, so we can use those for testing and will not need to provision more long-running clusters to excercise this enhancement.
+We can provision additional short-lived clusters in existing CI accounts if we want to provide additional end-to-end testing.
+
+[api-pull-request]: https://github.com/openshift/api/pull/326
```

Comment:
This pull request is now closed.  So may be we can remove it from here. Not sure what is the right action here.

Reply:
> This pull request is now closed.

It was closed pending this enhancement discussion.  If the enhancement lands, I'll re-open and reroll the PR.